src/ast/mod.rs

```diff
@@ -7357,6 +7354,102 @@ impl Display for UtilityOption {
     }
 }
 
+/// Represents the different options available for a SHOW <OBJECT>
+/// statement to filter the results. Example from Snowflake:
+/// https://docs.snowflake.com/en/sql-reference/sql/show-tables
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+pub struct ShowStatementOptions {
```

Comment:
Oh one thing actually, we might be missing the serde and visit derive attributes on some of the new structs?